revolut/__init__.py

```diff
@@ -166,6 +166,66 @@ def transaction(self, id):
     # TODO: def transaction_by_request_id(self, request_id):
 
 
+class ClientMerchant(utils._SetEnv):
+    live = False
```

Comment:
Tu raczej będzie `True` albo w ogóle to wywalimy jeśli nie będzie się dało sprawdzić czy jesteśmy na sandbox czy na prod.

Reply:
Aktualnie do wyrzucenia i do dodania jak otrzymamy klucz od Przemka i będą one rozróżnialne w jakiś sposób.